cmd/nephe-controller/main.go

```diff
@@ -68,6 +71,8 @@ func main() {
 	logging.SetDebugLog(enableDebugLog)
 	ctrl.SetLogger(logging.GetLogger("setup"))
 
+	opts.complete()
```

Comment:
if you fail to parse config, you should consider failing pod status to notReady

Reply:
Solved